infra-templates/k8s/45/docker-compose.yml.tpl

```diff
@@ -67,6 +67,7 @@ kubelet:
         - rancher-cni-driver:/etc/cni:ro
         - rancher-cni-driver:/opt/cni:ro
         - /dev:/host/dev:rprivate
+        - /usr/libexec/kubernetes/kubelet-plugins:/usr/libexec/kubernetes/kubelet-plugins
```

Comment:
Instead of adding this bind-mount here, add e.g. `--volume-plugin-dir=/var/lib/kubelet/volumeplugins` to the command line of `kubelet` above, e.g. line 34. Kubelet will use `/var/lib/kubelet/volumeplugins` instead and that directory has already been bind-mounted.